osu.Game/Skinning/Components/BeatmapAttributeText.cs

```diff
@@ -34,8 +38,27 @@ public partial class BeatmapAttributeText : Container, ISkinnableDrawable
         [SettingSource("Template", "Supports {Label} and {Value}, but also including arbitrary attributes like {StarRating} (see attribute list for supported values).")]
         public Bindable<string> Template { get; set; } = new Bindable<string>("{Label}: {Value}");
 
+        [SettingSource("Adjust Difficulty", "Should Mods that change Beatmap Statistics be applied to the shown Stats?")]
+        public BindableBool DifficultyAdjust { get; set; } = new BindableBool(true);
```

Comment:
Should not be a setting. Either this PR is the wrong direction entirely and none of this is necessary, or the component as it is in `master` is plain wrong and this diff is required. Having this toggleable helps nobody.

Reply:
This was briefly [discussed in discord](https://discord.com/channels/188630481301012481/188630652340404224/1059872767555682405) today, and either this setting should not exist (and all values should be post-mod application), or there should be 2 sets of variables offered for pre- and post-mod values.

Reply:
Well yes. The current implementation would probably have to somewhat improve as it is currently mighty inefficient and would only get worse. But it covers more use cases than the switch.